test/scripts/utils.test.js

```diff
@@ -25,24 +25,6 @@ describe('Libs', () => {
     expect(libs).to.equal('https://foo--milo--adobecom.hlx.live/libs');
   });
 
-  it('Supports milo stage libs with stage as host', () => {
```

Comment:
adding a `it.skip` with a comment might be useful here

Reply:
Going to merge this, as the stage code in our set libs is more important to remove. I can add an issue to add test skips if you think that is pertinent @Brandon32.